src/adapters/scene_category.js

```diff
@@ -137,13 +147,23 @@ export default class SceneCategory {
     });
   }
 
+  setPoiFeatureState = (id, state) => {
+    this.map.setFeatureState({ id, source: DYNAMIC_POIS_LAYER }, state);
+  }
+
   highlightPoiMarker = (poi, highlight) => {
+    if (this.hoveredPoi) {
```

Comment:
`this.hoveredPoi` does not seem to be use anywhere except in this `highlightPoiMarker` function, did I miss its use cases ?

Reply:
We need to store it to be able to remove the hovered effect on it.